### PR TITLE
D8 un 588

### PR DESCRIPTION
**Pull request by soda736**
Search page templates, preprocessing and regions.

### CODE REVIEW

File: web/sites/niauditoffice/themes/niauditoffice_theme/niauditoffice_theme.theme [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  // A list of search page titles on the Uregni site.
+  $search_pages = [
+    'News',
+    'Publications',
+    'Search',
+  ];
+
+  // If a search page set the variable to true.
+  foreach ($search_pages as $search_page) {
+    if ($variables['page']['#title'] == $search_page) {
```
Comment (omahm) [inline web/sites/niauditoffice/themes/niauditoffice_theme/niauditoffice_theme.theme:41]: This works but take a look at https://www.php.net/manual/en/function.in-array.php
Could replace 5 lines with 1.

**Review by ooneill547 — APPROVED** [review]

File: web/sites/niauditoffice/themes/niauditoffice_theme/templates/views/views-view-unformatted--search-result-page.html.twig [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * @see template_preprocess_views_view_unformatted()
+ */
+#}
+{% if title %}
+  <h3>{{ title }}</h3>
+{% endif %}
+{% for row in rows %}
+  {%
+    set row_classes = [
```
Comment (omahm) [inline web/sites/niauditoffice/themes/niauditoffice_theme/templates/views/views-view-unformatted--search-result-page.html.twig:23]: row_classes will be set with every look of the rows, I'd create this outside the loop

**Review by omahm — CHANGES_REQUESTED** [review]
Looks good overall. A few wee suggestions..  ping me if you want to go over in_array

File: web/sites/niauditoffice/themes/niauditoffice_theme/templates/views/views-view-unformatted--search-result-page.html.twig (hunk 2 above)
Comment (soda736) [reply web/sites/niauditoffice/themes/niauditoffice_theme/templates/views/views-view-unformatted--search-result-page.html.twig:23]: I am trying to add the search result class to every article in the view. There is code for this in the other unity sites and styling in place for the .search-results. Would it be better to have the search result class outside of the rows and add the styling to .search-results article?

File: web/sites/niauditoffice/themes/niauditoffice_theme/niauditoffice_theme.theme (hunk 1 above)
Comment (soda736) [reply web/sites/niauditoffice/themes/niauditoffice_theme/niauditoffice_theme.theme]: Think I know what you mean, I've made a change to it.

**Review by omahm — APPROVED** [review]
Yup, that's it. Good stuff

**Pull request merged**